geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -45,6 +45,11 @@ Create async-event-queues, disk-stores, gateway receivers, gateway senders, inde
 
     Create an index that can be used when executing queries.
 
+-   **[create jndi-binding](#create_jndi-binding)**
+
+    Create a JNDI binding that specifies a resource accessed with
+a JDBC connection.
```

Comment:
I like the wording as in "Create a JNDI binding that specifies resource attributes which describe a JDBC connection."